index.js

```diff
@@ -169,12 +176,16 @@ module.exports = class Hyperswarm extends EventEmitter {
     })
     this._allConnections.add(conn)
 
+    this.stats.connections.attempted++
+
     this.connecting++
     this._clientConnections++
     let opened = false
 
     conn.on('open', () => {
       opened = true
+      this.stats.connections.opened++
```

Comment:
should also track server connections ya?

Reply:
Good point, fixed

Reply:
Note: could also track client- and server connections separately, but I put them together for now

Reply:
Now it's tracking them separately (cleaner that way)